src/index.html

```diff
@@ -8,6 +8,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="card img"
+        class="card__image"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__description">Product code: 195434</p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price-wraper">
+        <p class="card__price-text">Price:</p>
+        <span class="card__price-amount">$2,199</span>
+      </div>
+      <a
```

Comment:
See [checklist №3](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md), you need to add empty lines here.
```suggestion
      </div>
      
      <div class="card__price-wraper">
        <p class="card__price-text">Price:</p>
        <span class="card__price-amount">$2,199</span>
      </div>
      
      <a
```